src/pyhf/optimize/opt_tflow.py

```diff
@@ -8,6 +8,11 @@
 class tflow_optimizer(AutoDiffOptimizerMixin):
     """Tensorflow Optimizer Backend."""
 
+    def __init__(self, *args, **kwargs):
+        """Tensorflow based optimizer based on autograd."""
+        self.lr = kwargs.get('lr', 1.0)
```

Comment:
what does lr stand for?

Reply:
learning late

Reply:
`lr` seems to be the standard for learning rate across TensorFlow and PyTorch too.